install/kubernetes/helm/meshery/charts/meshery-istio/templates/clusterrole.yaml

```diff
@@ -18,4 +18,5 @@ rules:
   - create
   - update
   - patch
-  - delete
\ No newline at end of file
+  - delete
+  - approve
```

Comment:
@utkarsh-pro I don't think this is necessary, but I also don't consider it harmful. I don't think that the adapter charts are individually used like this, are they @Aisuko?